lib/auth/auth_with_roles_test.go

```diff
@@ -6537,6 +6537,29 @@ func TestSafeToSkipInventoryCheck(t *testing.T) {
 	}
 }
 
+func TestPing_AccessRequestFeature(t *testing.T) {
+	ctx := context.Background()
+	srv := newTestTLSServer(t)
+
+	ping, err := srv.Auth().Ping(ctx)
+	require.Nil(t, err)
+	require.False(t, ping.ServerFeatures.AccessRequests.Enabled)
```

Comment:
Please add a custom message when using methods like require.False, the error it prints is borderline useless (something is the lines of "true is not false").